### ISSUE TITLE
unable to use SoapService as provider

### ISSUE THREAD

**Issue by ImPravin** [autopulous/angular2-soap#12]
As in example, found that `providers: [SoapService]` used in `user.authenticator.ts`
So I did same in my demo application, but getting error like 
"Uncaught TypeError: (void 0) is not a function", source: file:///android_asset/www/build/main.js (101810)" in android emulator 
& while checked on chrome console I found below error
Failed to load resource: the server responded with a status of 404 (Not Found) xdom.js
Failed to load resource: the server responded with a status of 404 (Not Found) xdom2jso.js

Please help me out for this problem.

**Comment by MA-Maddin** [comment]
Add this to your `angular-cli.json`:

```
{
  // ...
  "apps": [
    {
      // ...
      "scripts": [
        "../node_modules/autopulous-angular2-soap/vendor/autopulous-xdom/xdom.js",
        "../node_modules/autopulous-angular2-soap/vendor/autopulous-xdom2jso/xdom2jso.js"
      ],
    }
  ]
}
```

**Comment by martenwallewein** [comment]
Same problem here, any solutions?

**Comment by broweratcognitecdotcom** [comment]
I don't think this is optimally maintained. I had to make changes, but I did get this working reliably in newer angular projects. But we have our own version. It has been a while but I think I just copied and pasted the code from those 2 files into the main file and made some small changes to get that to work.
